io2-portal/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.0.3-rc.13
+version: 1.0.3-rc.14
```

Comment:
As I said in slack, you need to also bump the chart version so we can deploy this specific version only.